csvtools.nim

```diff
@@ -334,7 +334,7 @@ proc writeToCsv*[T](ts: openarray[T], f: var File, separator = ',',
     f.write(line)
 
 proc writeToCsv*[T](ts: openarray[T], path: string, separator = ',',
-  quote = '\"'; escape = '\"'; quoteAlways = false; dateLayout: string = "") =
+  quote = '\"'; escape = '\"'; quoteAlways = false; dateLayout: string = "", writeHeaders: bool = true) =
```

Comment:
To avoid changing the default behaviour, I think that `writeHeaders` should default to false

Reply:
changed to false